### PR TITLE
Change POST API error response from html to json format

### PR DESCRIPTION
**Pull request by chintanrp**

### CODE REVIEW

File: pyServer/AzureDiskInspectService.py [hunk 1]
```diff
@@ -663,7 +680,10 @@ def do_POST(self):
             failureStatusText = 'Server Error'
         finally:
             if (not requestSucceeded):
-                self.send_error(failureResultCode, "%s: %s" % (failureStatusText, str(telemetryException)))
+                # Change error response default format from html to json
+                self.error_message_format = json.dumps(ErrorDetails("%(explain)s", "%(message)s").__dict__)
```
Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:684]: Maybe add a link to the reference doc on how the error message formatter works.

File: pyServer/AzureDiskInspectService.py [hunk 2]
```diff
@@ -85,6 +85,14 @@ def getContainerId():
 class ResponseHeaderMetadata:
     KEEPALIVETHREAD_TIMEOUT_IN_MINS = "KeepAliveThread-Timeout-In-Mins"
 
+"""
+Error response model
+"""
```
Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:90]: Add some details like: "Error response entity model in compliance with ARM RPC protocol..." etc

File: pyServer/AzureDiskInspectService.py (hunk 1 above)
Comment (chintanrp) [reply pyServer/AzureDiskInspectService.py:685]: Added reference doc link.

File: pyServer/AzureDiskInspectService.py (hunk 2 above)
Comment (chintanrp) [reply pyServer/AzureDiskInspectService.py:90]: Updated comment as suggested

**Review by waynekuo — APPROVED** [review]
:shipit:

**Pull request merged**